src/main/java/com/example/springboot/dao/BookDAOService.java

```diff
@@ -0,0 +1,148 @@
+package com.example.springboot.dao;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+import org.springframework.stereotype.Repository;
+
+import com.example.springboot.common.Book;
+import com.mysql.jdbc.Statement;
+
+/**
+ * This class implement methods from IBookDAOService interface
+ * @author Kohut Dmytro
+ * @version 1.0
+ */
+@Repository("iBookService")
+public class BookDAOService implements IBookDAOService {
```

Comment:
ny sho za blyat BookDAOService che ne BookJavaMatEgoClassDAOAsheBlaytNefigoviuService
prosto BookDAOImpl

Reply:
`BookDAOImpl` - bad name for the service. Don't listening this guy ;) `BookDAOService ` is a good name for Book DAO 
service